### PR TITLE
MMI Sprite refactor, new MMI sprites for weird brains

### PR DESCRIPTION
**Pull request by QuinnAggeler**
Refactors MMIs to refer to icon variables defined on the brain organs when assigning the filled sprites, so now species with weird brains (Drask and Slimes) have their own MMI sprites.
<a href="http://imgur.com/cz6Qn4i"></a>

:cl:HugoLuman
:refactor: MMI filled sprites are now dependent upon the ``mmi_icon`` and ``mmi_icon_state`` vars of brains
:rscadd: Adds sprites for Drask brains and slime cores inside MMIs
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/brain/MMI.dm [hunk 1]
```diff
@@ -49,6 +49,11 @@
 			icon = 'icons/mob/alien.dmi'
 			icon_state = "AlienMMI"
 			alien = 1
+		else if(istype(O,/obj/item/organ/internal/brain/drask))
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/brain/MMI.dm]: i don't expect you to change this



but istype lists for this sorta thing are aaaaaaaaaaaaaaaaaaaaaawful

Comment (QuinnAggeler) [inline code/modules/mob/living/carbon/brain/MMI.dm]: How should it be handled then? Should I refactor how the bit above does it while I'm here?

Comment (Crazylemon64) [inline code/modules/mob/living/carbon/brain/MMI.dm]: Give brains an `mmi_icon` var, which is used in MMI code to display how they look

**Comment by LittleBigKid2000** [comment]
While we're at it, maybe slime people should get an MMI sprite, too.

File: code/modules/mob/living/carbon/brain/MMI.dm (hunk 1 above)
Comment (RemieRichards) [inline code/modules/mob/living/carbon/brain/MMI.dm]: ^ literally the only correct solution.

Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/brain/MMI.dm]: Yup; please do as @Crazylemon64 and @RemieRichards suggested.

**Comment by QuinnAggeler** [comment]
@Fox-McCloud , it's a refactor again now, doing what @Crazylemon64 said. Though I had to leave in the type check for xeno brains.

**Comment by QuinnAggeler** [comment]
Crap, didn't mean to close it

**Comment by Fox-McCloud** [comment]
:+1:

**Comment by LittleBigKid2000** [comment]
You listened to me 
👍

**Pull request merged**